interfaces/policy/helpers.go

```diff
@@ -221,6 +221,36 @@ func checkSlotConnectionConstraints(connc *ConnectCandidate, cstrs []*asserts.Sl
 	return firstErr
 }
 
+func checkSnapTypeSlotInstallationConstraints(ic *InstallCandidateMinimalCheck, slot *snap.SlotInfo, cstrs *asserts.SlotInstallationConstraints) error {
```

Comment:
for consistency could we add a "1" to this name, checkSnapTypeSlotInstallationConstraints1